### ISSUE TITLE
Can't change a privacy option 

### ISSUE THREAD

**Issue by ghost** [searxng/searxng#1503]
**Version of SearXNG, commit number if you are using on master branch and stipulate if you forked SearXNG**
Searx.be instance 
**What happened?**
If I set "Query in the page's title" to Disabled and save, it automatically reverts to Enabled
**Expected behavior**
My selected option should be kept in the cookies and not revert

**Comment by unixfox** [comment]
I won't fix that because the oscar patched theme on my instance is in an unstable state but I'm waiting on https://github.com/searxng/searxng/issues/1169 to drop the oscar theme on my instance.

Closing as unfortunately there is no fix planned.
